CIFAR-100/train_with_distillation.py

```diff
@@ -116,11 +118,14 @@ def train_with_distill(d_net, epoch, alpha):
         loss_CE = criterion_CE(outputs, targets)
 
         loss = loss_CE + alpha * (loss_distill.sum()) / batch_size / 1000
+        loss_distill = loss_distill.sum() / batch_size / 1000
```

Comment:
```suggestion
        loss_CE = criterion_CE(outputs, targets)
        loss_distill = loss_distill.sum() / batch_size

        loss = loss_CE + alpha * loss_distill
```
Make code prettier, remove `/1000` and set default value of alpha smaller.